linode/manager
# change: [M3-8614] - Update Image Upload dropzone copy as part of Image Service Gen2

## Description 📝

Makes some minor copy updates and UI tweaks on the Image Upload page based on UX's request 🎨 

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-09-23 at 12 07 49 PM](https://github.com/user-attachments/assets/195b07ac-1157-49a8-97c9-05c622cf8ec2) | ![Screenshot 2024-09-23 at 12 06 57 PM](https://github.com/user-attachments/assets/6ecfa99e-eead-4654-8b9d-0341c1144b20) |

## How to test 🧪

- Go to http://localhost:3000/images/create/upload
- Check UI and verify the changes make logical sense 🧠 
- Check copy for spelling ✏️ 
- Check for any UI regressions 👁️ 

## As an Author I have considered 🤔
